### PR TITLE
casadm standby cache commands for tests

### PR DESCRIPTION
**Pull request by pdebski21**
New casadm commands for standby mode and change of naming convention following new design.

### CODE REVIEW

**Review by Deixx — CHANGES_REQUESTED** [review]

**Review by Deixx — CHANGES_REQUESTED** [review]
Don't include OCF in PRs which should not touch OCF

File: test/functional/api/cas/casadm.py [hunk 1]
```diff
@@ -42,6 +42,58 @@ def start_cache(cache_dev: Device, cache_mode: CacheMode = None,
...
+    )
+    if output.exit_code != 0:
+        raise CmdException("Failed to init standby cache.", output)
+    return Cache(cache_dev)
+
+
+def standby_load(cache_dev: Device, shortcut: bool = False,
+                 kernel_params: KernelParameters = KernelParameters()):
+    if kernel_params != KernelParameters.read_current_settings():
+        reload_kernel_module("cas_cache", kernel_params.get_parameter_dictionary())
```
Comment (Deixx) [inline test/functional/api/cas/casadm.py:67]: drop reloading kernel modules when loading cache (we don't want to change the settings when we're restoring the cache)

File: test/functional/api/cas/cli.py [hunk 2]
```diff
@@ -108,6 +108,37 @@ def start_cmd(cache_dev: str, cache_mode: str = None, cache_line_size: str = Non
     return casadm_bin + command
 
 
+def standby_init_cmd(cache_dev: str, cache_id: str, cache_line_size: str,
+                     force: bool = False, shortcut: bool = False):
+    command = " --standby --init "
```
Comment (Deixx) [inline test/functional/api/cas/cli.py:113]: cosmetic: remove the space after `--init` (all the params below have a preceding space)

File: test/functional/api/cas/cli.py [hunk 3]
```diff
@@ -108,6 +108,37 @@ def start_cmd(cache_dev: str, cache_mode: str = None, cache_line_size: str = Non
...
+    command += (" -d " if shortcut else " --cache-device ") + cache_dev
+    if cache_id is not None:
```
Comment (Deixx) [inline test/functional/api/cas/cli.py:115]: remove the condition (cache id is mandatory)

File: test/functional/api/cas/cli.py [hunk 4]
```diff
@@ -108,6 +108,37 @@ def start_cmd(cache_dev: str, cache_mode: str = None, cache_line_size: str = Non
...
+        command += (" -i " if shortcut else " --cache-id ") + cache_id
+    if cache_line_size is not None:
```
Comment (Deixx) [inline test/functional/api/cas/cli.py:117]: remove the condition (cache line size is mandatory)

File: test/functional/api/cas/cli.py [hunk 5]
```diff
@@ -108,6 +108,37 @@ def start_cmd(cache_dev: str, cache_mode: str = None, cache_line_size: str = Non
...
+        command += (" -x " if shortcut else " --cache-line-size ") + cache_line_size
+    if force:
+        command += " -f" if shortcut else " --force"
+
```
Comment (Deixx) [inline test/functional/api/cas/cli.py:121]: add `return casadm_bin + command`

File: test/functional/api/cas/cli.py [hunk 6]
```diff
@@ -108,6 +108,37 @@ def start_cmd(cache_dev: str, cache_mode: str = None, cache_line_size: str = Non
...
+
+def standby_load_cmd(cache_dev: str, shortcut: bool = False):
+    command = " --standby --load "
```
Comment (Deixx) [inline test/functional/api/cas/cli.py:124]: cosmetic: remove the space after `--load`

File: test/functional/api/cas/cli.py [hunk 7]
```diff
@@ -108,6 +108,37 @@ def start_cmd(cache_dev: str, cache_mode: str = None, cache_line_size: str = Non
...
+    command += (" -d " if shortcut else " --cache-device ") + cache_dev
+    return casadm_bin + command
+
+
+def standby_detach_cmd(cache_id: str, shortcut: bool = False):
+    command = " --standby --detach "
```
Comment (Deixx) [inline test/functional/api/cas/cli.py:130]: cosmetic: remove the space after `--detach`

File: test/functional/api/cas/cli.py [hunk 8]
```diff
@@ -108,6 +108,37 @@ def start_cmd(cache_dev: str, cache_mode: str = None, cache_line_size: str = Non
...
+    command += (" -i " if shortcut else " --cache-id ") + cache_id
+    return casadm_bin + command
+
+
+def standby_activate_cmd(cache_dev: str, cache_id: str, shortcut: bool = False):
+    command = " --standby --activate "
```
Comment (Deixx) [inline test/functional/api/cas/cli.py:136]: cosmetic: remove the space after `--activate`

**Review by Deixx — CHANGES_REQUESTED** [review]

File: test/functional/api/cas/casadm.py [hunk 9]
```diff
@@ -42,6 +42,56 @@ def start_cache(cache_dev: Device, cache_mode: CacheMode = None,
...
+            shortcut=shortcut,
+        )
+    )
+    if output.exit_code != 0:
+        raise CmdException("Failed to init standby cache.", output)
+    return Cache(cache_dev)
+
+
+def standby_load(cache_dev: Device, shortcut: bool = False,
+                 kernel_params: KernelParameters = KernelParameters()):
```
Comment (Deixx) [inline test/functional/api/cas/casadm.py:65]: remove kernel_params from args also ;)

**Review by Deixx — CHANGES_REQUESTED** [review]

**Review by Deixx — APPROVED** [review]

**Review by robertbaldyga — APPROVED** [review]

**Pull request merged**